pcsx2/R3000AInterpreter.cpp

```diff
@@ -39,9 +39,9 @@ void psxBGEZ()         // Branch if Rs >= 0
 
 void psxBGEZAL()   // Branch if Rs >= 0 and link
 {
+	_SetLink(31);
 	if (_i32(_rRs_) >= 0)
 	{
-		_SetLink(31);
 		doBranch(_BranchTarget_);
```

Comment:
why not use ``_SetLink`` ``after doBranch()`` to have it follow the order as mentioned on comment ?